api-tests/helpers.go

```diff
@@ -59,7 +59,7 @@ func TestString(t TestingT, name string) string {
 	n, err := rand.Int(rand.Reader, big.NewInt(math.MaxInt32))
 	require.NoError(t, err)
 
-	return strings.ReplaceAll(fmt.Sprintf("pmm-api-tests-%s-%s-%s-%d", Hostname, t.Name(), name, n), "/", "-")
+	return strings.ReplaceAll(fmt.Sprintf("api-test-%s-%s-%s-%d", Hostname, t.Name(), name, n), "/", "-")
```

Comment:
Do we really need that much unique things? I think we can safely drop Hostname

Reply:
Yeas, I was thinking about it too, but I wasn't sure if that would break things. I'll give it a try anyway :)

Reply:
The API tests passed with that change :)

https://pmm.cd.percona.com/job/pmm3-api-tests/936/console

Done.